### ISSUE TITLE
The plugin does not respect *.yml

### ISSUE THREAD

**Issue by jhcook** [vagrant-dotvm/vagrant-dotvm#35]
After trying to use *.yml files, I received an error of:

```
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
There are errors in the configuration of this machine. Please fix
the following errors and try again:

vm:
* A box must be specified.
```

After renaming the files to *.yaml this problem went away. The error was not indicative of the problem, either.

**Comment by krzysztof-magosa** [comment]
The official documentation of YAML says that correct extension is `.yaml`.
DotVM parse only `.yaml` files, so if you don't have them Vagrant is unconfigured.

**Comment by jhcook** [comment]
I understand that, but yml is used quite significantly. Should we be terse or since it’s commonly used, go ahead and provide support?

http://stackoverflow.com/questions/21059124/is-it-yaml-or-yml

-- 

Justin Cook 

On 4 March 2016 at 12:23:43, Krzysztof Magosa (notifications@github.com) wrote:

The official documentation of YAML says that correct extension is .yaml.
DotVM parse only .yaml files, so if you don't have them Vagrant is unconfigured.

**Comment by krzysztof-magosa** [comment]
I've added support for `.yml` in aa23f02059f932a525d23ff91b5187dadbb8a2d0.
Anyway I think that should be better to promote only one extension (`.yaml`), for consistency.

**Comment by krzysztof-magosa** [comment]
To be released in 0.38.0
